packages/babel-plugin-transform-react-jsx-development/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@babel/plugin-transform-react-jsx-development",
```

Comment:
Why not an option in the main transform? :thinking:

Reply:
We decided that because development mode (jsxDEV) is sufficiently different enough from production mode (jsx) in both arguments and maybe potential future optimization, it would be better to make the split more explicit. This might also simplify Preact's use case if they only want to use `jsxDEV`